spack_config/toss_4_x86_64_ib_cray/spack.yaml

```diff
@@ -553,6 +845,8 @@ spack:
         - cray-libsci/24.03.0
         - cray-mpich/8.1.29
         - craype-network-ofi
+      - spec: "cray-libsci@24.11.0%rocmcc+mpi~openmp+shared"
+        prefix: /opt/cray/pe/libsci/24.11.0/AMD/6.0/x86_64
```

Comment:
I am interested to know where you find information regarding the libsci version to use. Also, I'd like a note if this is MFEM specific or generic (would end user use the same, or just the MFEM team) ?

Reply:
This is the version that is currently loaded by default on `tioga` -- that's why I switched to it. I think it's also the latest version that is installed there. I use directly the "AMD version" because that's what you're supposed to use with the `rocmcc` compilers.